packages/react/src/components/TableOfContents/README.md

```diff
@@ -67,10 +80,9 @@ DDS_TOC=true
 
 ## menuItems
 
-| Properties Name | Data Type | Description     |
-| --------------- | --------- | --------------- |
-| title           | String    | Menu title text |
-| id              | String    | Menu id         |
+| Name    | Data Type | Default value                                  | Description |
+| ------- | --------- | ---------------------------------------------- | ----------- |
+| `theme` | String    | `` | Set the dark theme if the value is `g100` |
```

Comment:
Can you create another table for theme instead and keep the original `menuItems` table?

like https://github.com/carbon-design-system/carbon-for-ibm-website/pull/149

Oh and add `theme` prop to the `Props` table above with the corresponding data (required, data type, default value, description)